build.gradle

```diff
@@ -81,16 +82,24 @@ static def configurePublishingRepositories(repositoryHandler) {
     }
 }
 
-publishing {
-    publications {
-        fat(MavenPublication) {
-            artifact jar
-            artifact shadowJar
-            artifact deobfJar
+if (!System.env.RELEASE) {
+    build.dependsOn ':forge:publishToMavenLocal'
```

Comment:
See comments in the related PR, [AntimatterAPI#81](https://github.com/GregTech-Intergalactical/AntimatterAPI/pull/81)